buildo/github-workflow-pal
# #72: add `only if nobody answer before` option to elephant reminder (closes #72)

Issue #72

## Test Plan
- Add a reminder 'only if nobody answer before'

![image](https://cloud.githubusercontent.com/assets/3280300/16015831/e15eb6ee-3197-11e6-9443-3fd4f80573c0.png)

- you should see

![image](https://cloud.githubusercontent.com/assets/3280300/16015844/f20ee39c-3197-11e6-9026-3b9a613ded9d.png)

- if you refresh the page, you should see the same
- add a comment to the issue/pr, and refresh
- you shouldn't see the reminder set anymore

- Add a reminder 'always'

![image](https://cloud.githubusercontent.com/assets/3280300/16015891/1e7e5c82-3198-11e6-8e02-776bfa7cc157.png)

- you should see

![image](https://cloud.githubusercontent.com/assets/3280300/16015900/27c7846c-3198-11e6-9c02-c5242505f76f.png)

- if you refresh the page, you should see the same
- add a comment to the issue/pr, and refresh
- you should still see the reminder
